app/src/main/java/org/mozilla/fenix/tabstray/NavigationInteractor.kt

```diff
@@ -146,6 +147,8 @@ class DefaultNavigationInteractor(
     }
 
     override fun onShareTabs(tabs: Collection<TabSessionState>) {
+        TabsTray.shareSelectedTabs.record(TabsTray.ShareSelectedTabsExtra(tabCount = tabs.size))
```

Comment:
The convention here is to track through the `GleanMetricsService`.  Is there a reason that we're skipping that?

Reply:
The tracking is unnecessary and it also might have some performance issues (see https://github.com/mozilla-mobile/fenix/issues/21292). There are no advantages in using excessive wrapping and this direct approach has been used successfully in Focus telemetry migration. IMO it might be a good idea to also move to the new calling API and implement new telemetry in this way. (Also see https://github.com/mozilla-mobile/fenix/issues/19967)

Reply:
I see and it's a good point.  But can we continue to use the wrapping until we decide to remove it? This way we don't have a mishmash of different ways to implement telemetry.

Reply:
As discussed, since we're moving towards removing the wrapper, I think this is good to go.  Thanks!